boards/mips-malta/Makefile.include

```diff
@@ -0,0 +1,3 @@
+export CPU = mips32r2_common
+export INCLUDES += -I$(RIOTBOARD)/$(BOARD)/include/
+export CFLAGS += -mhard-float -mdsp
```

Comment:
How about a simple variable like `USE_HARD_FLOAT` and then checking that in the `mips_common/Makefile.include` and set the CFLAGS there?